recipe/meta.yaml

```diff
@@ -1,6 +1,7 @@
 {% set name = "towncrier" %}
-{% set version = "19.9.0" %}
-{% set sha256 = "19916889879353a8863f3de8cb1ef19b305a0b5cfd9d36159d76ca2fef08e9aa" %}
+{% set version = "21.3.0" %}
+{% set build_number = 1 %}
```

Comment:
Better to remove this line and edit directly the build number line 16. At the moment, the variable `build_number` is not being used and therefore the build number is still 0 as defined line 16 - you check in the build log on azure pipelines.

Reply:
Moved. sorry about that.